modules/cli/src/main/scala/scala/cli/commands/new/New.scala

```diff
@@ -0,0 +1,19 @@
+package scala.cli.commands.`new`
+
+import caseapp.core.RemainingArgs
+import giter8.Giter8
+
+import scala.build.Logger
+import scala.cli.commands.ScalaCommand
+import scala.cli.commands.shared.HelpCommandGroup
+
+object New extends ScalaCommand[NewOptions] {
+  override def group: String = HelpCommandGroup.Main.toString
+
+  override def scalaSpecificationLevel = SpecificationLevel.IMPLEMENTATION
+
+  override def runCommand(options: NewOptions, remainingArgs: RemainingArgs, logger: Logger): Unit =
+    val inputArgs = remainingArgs.remaining
+    Giter8.run(inputArgs.toArray)
+    ()
```

Comment:
I should probably handle the exitcode giter8 gives back better, not sure what is idiomatic for scala-cli

Reply:
Hi @zetashift, thank you for your contribution. The overall changes are great, and `scala-cli` team would be happy to merge your changes.

We suggest downloading `giter8` as a JAR and running it using the Java command instead of adding the `giter8` dependency directly to the `scala-cli` code.

Our intention is to keep the current size of scala-cli, which is relatively small. We prefer not to add new dependencies that would significantly increase the size of the `scala-cli` fat JAR, which in this case add about 3-4Mb.

You can take inspiration from this [example](https://github.com/VirtusLab/scala-cli/blob/d062ea14f3286adbad722117bbac45cb69284085/modules/cli/src/main/scala/scala/cli/launcher/LauncherCli.scala#L33). In that case, scala-cli downloads a nightly version of the [scala-cli JVM launcher](https://scala-cli.virtuslab.org/docs/commands/basics/#scala-cli-version) and runs it.
 
Version, organization, and name of `giter8` can be hardcoded in `Constants.scala`, similar to how it was done for [scalafmt](https://github.com/VirtusLab/scala-cli/blob/d062ea14f3286adbad722117bbac45cb69284085/build.sc#L403).